tests/test_adobe_vendor_id.py

```diff
@@ -624,6 +625,38 @@ def setup_method(self):
             },
         )
 
+    @pytest.mark.parametrize(
+        "registration_status, authdata_utility_type",
+        [
+            (RegistrationConstants.SUCCESS_STATUS, AuthdataUtility),
+            (RegistrationConstants.FAILURE_STATUS, type(None)),
+            (None, type(None)),
+        ],
+    )
```

Comment:
New test to verify that we don't get an AuthdataUtility unless a successful registration has been accomplished.